tests/features/ecas-register.feature

```diff
@@ -1,11 +1,44 @@
+@api @casMockServer
 Feature: Register through OE Authentication
   In order to be able to have new users
   As an anonymous user of the system
   I need to be able to go to the registration URL
 
+  Background:
+    Given CAS users:
+      | Username    | E-mail                            | Password           | First name | Last name | Department    | Organisation |
+      | jb007       | 007@mi6.eu                        | shaken_not_stirred | James      | Bond      | DIGIT.A.3.001 | external     |
```

Comment:
This create the user for all tests. It shouldn't be like this. Move it to the test that really needs it.